WorksheetB

```diff
@@ -0,0 +1,39 @@
+# Fallout word guessing game
```

Comment:
Flowchart looks OK, but is missing a check for the maximum number of guesses being exceeded.